bot/functions/joinAutorole.ts

```diff
@@ -0,0 +1,9 @@
+import { Client, Snowflake } from "discord.js";
+
+export function setupAutorole(client: Client, roleId: Snowflake){
+    client.on('guildMemberAdd', async (member) => {
+        const role = await member.guild.roles.fetch(roleId);
+        if(role == null) return;
```

Comment:
Raczej unikaj `==` i używaj `===`. Podwójna równość dodatkowo konwertuje sobie typ, co daje dziwne efekty, szczególnie jeśli nie zna się tego algorytmu według którego ona to robi (a mało kto go zna). Jeśli potrzebujesz porównać coś o różnych typach, zrób konwersję samemu.

Reply:
Okej dzieki